libs/base/ui/action/color-mode-selector/src/color-mode-selector.component.ts

```diff
@@ -0,0 +1,87 @@
+import { i18n, rootInjectable } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { state } from 'lit/decorators.js';
+import { styles } from './color-mode-selector.styles';
+import { ModeEvent, toggleMode } from './utilities';
+
+export const EVENT_TOGGLE_COLOR = 'oryx.toggle-mode';
+
+export class ColorModeSelectorComponent extends LitElement {
+  static styles = [styles];
+
+  protected darkMode = 'mode-dark';
+  protected lightMode = 'mode-light';
```

Comment:
we have a few other locations where we need to distinquish dark vs light mode. can we centralise this model inside ui or experience package? We have some UI components that could benefit from shared model, as well as the color generation stuff that would. Might be something for a follow up when we move theme/color out of core.

Reply:
If we have in plans move out `theme` from core I'll move them into it